### PR TITLE
Navbar: link my requests to navbar

### PR DESCRIPTION
**Pull request by Pineirin**
Screenshot from 2022-02-16 15-39-40
closes: inveniosoftware/invenio-app-rdm#1255

### CODE REVIEW

**Comment by Pineirin** [comment]
I'm a bit unsure of the underline under the icon. Should I maybe remove it?

File: invenio_app_rdm/records_ui/views/__init__.py [hunk 1]
```diff
@@ -79,11 +80,17 @@ def create_blueprint(app):
         routes["dashboard_home"],
         view_func=dashboard,
     )
+
     blueprint.add_url_rule(
         routes["dashboard_item"],
         view_func=dashboard,
     )
 
+    blueprint.add_url_rule(
```
Comment (zzacharo) [inline invenio_app_rdm/records_ui/views/__init__.py:89]: This is not needed as the `routes["dashboard_item"]` handles the dashboard request.

File: invenio_app_rdm/theme/views.py [hunk 2]
```diff
@@ -41,6 +41,12 @@ def init_menu():
             order=1
         )
 
+        current_menu.submenu('requests.deposit').register(
```
Comment (zzacharo) [inline invenio_app_rdm/theme/views.py:44]: Not needed I think as well

File: invenio_app_rdm/records_ui/views/deposits.py [hunk 3]
```diff
@@ -294,6 +294,16 @@ def dashboard(dashboard_name=None):
     )
 
 
+@login_required
+def dashboard_requests():
```
Comment (zzacharo) [inline invenio_app_rdm/records_ui/views/deposits.py:298]: Not needed as well :)

File: invenio_app_rdm/theme/views.py [hunk 4]
```diff
@@ -41,6 +41,12 @@ def init_menu():
...
+            'invenio_app_rdm_records.dashboard_requests',
+            _('My dashboard'),
```
Comment (zzacharo) [inline invenio_app_rdm/theme/views.py:46]: You can use here the `invenio_app_rdm_records.dashboard` endpoint and pass this argument https://github.com/inveniosoftware/flask-menu/blob/master/flask_menu/__init__.py#L112  i.e 
```
endpoint_arguments_constructor=lamda _: dict(route="/requests")
```

**Review by zzacharo — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]

**Pull request merged**